packages/shared/src/components/cards/ReferralWidget.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { ReactElement } from 'react';
+import { Button, ButtonSize } from '../buttons/Button';
+import {
+  getFacebookShareLink,
+  getTwitterShareLink,
+  getWhatsappShareLink,
+} from '../../lib/share';
+import WhatsappIcon from '../icons/Whatsapp';
+import FacebookIcon from '../icons/Facebook';
+import TwitterIcon from '../icons/Twitter';
+import { TextField } from '../fields/TextField';
+import { ReferralCampaignKey, useReferralCampaign } from '../../hooks';
+import { link } from '../../lib/links';
+import { useShareOrCopyLink } from '../../hooks/useShareOrCopyLink';
+import { labels } from '../../lib';
+import { AnalyticsEvent, TargetId } from '../../lib/analytics';
+import { SimpleTooltip } from '../tooltips';
+
+const ReferralWidget = (): ReactElement => {
+  const { isReady, url } = useReferralCampaign({
+    campaignKey: ReferralCampaignKey.Generic,
+  });
+  const inviteLink = url || link.referral.defaultUrl;
+  const [, onShareOrCopyLink] = useShareOrCopyLink({
+    text: labels.referral.generic.inviteText,
+    link: inviteLink,
+    trackObject: () => ({
+      event_name: AnalyticsEvent.CopyReferralLink,
+      target_id: TargetId.ProfilePage,
+    }),
+  });
+  const onShareClick = () => {
+    onShareOrCopyLink();
+  };
+
+  return (
+    <div className="flex laptopL:absolute flex-col tablet:flex-row laptopL:flex-col flex-wrap justify-between p-4 laptopL:m-6 mt-6 laptopL:mt-0 mb-4 laptopL:max-w-widget laptopL:h-auto rounded-2xl border border-theme-divider-tertiary laptopL:left-[40rem] max-w-fit bg-theme-bg-primary">
+      <h3 className="mb-2 font-bold typo-title3">Invite friends</h3>
+      <p className="contents text-salt-50 typo-callout">
```

Comment:
We don't support named colors.
```suggestion
      <p className="contents text-theme-label-secondary typo-callout">
```